_examples/maps/test.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2017 The go-python Authors.  All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from __future__ import print_function
+import maps
+
+a = maps.NewMap()
+b = {1: 3.0, 2: 5.0}
+print('maps.MapsFunc from Go map:', maps.MapsFunc(a))
+print('maps.MapsFunc from Python dictionary:', maps.MapsFunc(b))
+
```

Comment:
please add 2 more tests:

- one with a python dict whose elements are strings (and try to pass it to `maps.Func` and check we correctly detect/handle that)
- one with a Go map whose elements are also strings (and try to pass it to `maps.Func`)